client/spring-zeebe/src/main/java/io/camunda/zeebe/spring/client/bean/value/ZeebeDeploymentValue.java

```diff
@@ -8,11 +8,14 @@
 
   private List<String> classPathResources;
 
+  private List<String> bpmnLocations;
```

Comment:
I don#t think we need two attributed - "resources" should be sufficient - and the old "classPathResources" can be translated into the right Spring resources string

Reply:
I see your point.